examples/ct/instance.go

```diff
@@ -82,6 +84,21 @@ func LogConfigFromFile(filename string) ([]LogConfig, error) {
 	return cfg, nil
 }
 
+var stringToKeyUsage = map[string]x509.ExtKeyUsage{
+	"Any":                        x509.ExtKeyUsageAny,
+	"ServerAuth":                 x509.ExtKeyUsageServerAuth,
+	"ClientAuth":                 x509.ExtKeyUsageClientAuth,
+	"CodeSigning":                x509.ExtKeyUsageCodeSigning,
+	"EmailProtection":            x509.ExtKeyUsageEmailProtection,
+	"IPSECEndSystem":             x509.ExtKeyUsageIPSECEndSystem,
+	"IPSECTunnel":                x509.ExtKeyUsageIPSECTunnel,
+	"IPSECUser":                  x509.ExtKeyUsageIPSECUser,
+	"TimeStamping":               x509.ExtKeyUsageTimeStamping,
+	"OCSPSigning":                x509.ExtKeyUsageOCSPSigning,
+	"MicrosoftServerGatedCrypto": x509.ExtKeyUsageMicrosoftServerGatedCrypto,
+	"NetscapeServerGatedCrypto":  x509.ExtKeyUsageNetscapeServerGatedCrypto,
+}
+
 // SetUpInstance sets up a log instance that uses the specified client to communicate
 // with the Trillian RPC back end.
 func (cfg LogConfig) SetUpInstance(client trillian.TrillianLogClient, deadline time.Duration) (*PathHandlers, error) {
```

Comment:
Could you add `TODO(daviddrysdale): add unit tests` at the top of this function while you're in here?